### ISSUE TITLE
Problem with AuditEvent date's offset

### ISSUE THREAD

**Issue by riccardos77** [thepirat000/Audit.NET#287]
**Describe the bug**
In AuditEvent StartDate and EndDate are in Utc but Kind has value Unspecified. 
Because of this I have problems when I try to convert the date back to DateTimeOffset as it is considered local.

**To Reproduce**
Look at value of Kind property in StartDate and EndDate.

**Expected behavior**
Both StartDate and EndDate must have Kind=Utc 
A possibile solution is to use something like:
```csharp
_event.StartDate = new DateTime(Configuration.SystemClock.UtcNow.Ticks, DateTimeKind.Utc)
```
here
https://github.com/thepirat000/Audit.NET/blob/c4e2eadc443a30841ca32f72ff09210c26838870/src/Audit.NET/AuditScope.cs#L52

**Libraries (specify the Audit.NET extensions being used including version):**
For example:
 - Audit.NET: 15.0.5

**Additional context**
a further improvement could be the addition of a new property in the ISystemClock interface such as UtcNowDate in order to return an already correctly created UTC date and use wherever it is needed.

**Comment by thepirat000** [comment]
I think I'll change the ISystemClock property to be a `DateTime` instead, so the Kind is preserved

**Comment by thepirat000** [comment]
This was fixed on version 15.2.0, please upgrade your references and re-test.
